com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java

```diff
@@ -342,7 +342,7 @@ protected ModVisitor makeModVisitor(CGNode n, Collection<PointerKey> result, Poi
     return makeModVisitor(n, result, pa, h, false);
   }
 
-  protected ModVisitor<T,? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
+  protected ModVisitor<T, ? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
```

Comment:
This was [just changed](https://github.com/wala/WALA/pull/172/files#diff-409e334c66c9e19b63fba57e6cb66973R345) by @khatchad to remove the `? extends`.  @khatchad does the wildcard cause a problem?